pkg/keys/keystore_test.go

```diff
@@ -154,7 +154,7 @@ var _ = Describe("Create Private Key", func() {
 
 		client := k8sfake.NewSimpleClientset(kubeobjects...)
 
-		returnedPrivateKey, err := GetOrCreatePrivateKey(client, namespace, secret)
+		returnedPrivateKey, err := GetOrCreatePrivateKey(client, namespace, secret, map[string]string{})
```

Comment:
Can you add a test that verifies that if you pass some labels, they are actually used?

Reply:
Done in `auth-config_test.go`